coffeescript-loader/loader.js

```diff
@@ -10,7 +19,7 @@ const baseURL = pathToFileURL(process.cwd() + '/').href;
 // CoffeeScript files end in .coffee, .litcoffee or .coffee.md.
 const extensionsRegex = /\.coffee$|\.litcoffee$|\.coffee\.md$/;
 
-export function resolve(specifier, context, defaultResolve) {
+export async function resolve(specifier, context, defaultResolve) {
```

Comment:
I wavered on this a few times, but decided, since this would be an official example, it should have the `async` to remind the user that hooks are async, regardless of whether they use async.